src/crawlee/storages/_request_list.py

```diff
@@ -91,37 +85,3 @@ async def mark_request_as_handled(self, request: Request) -> None:
     @override
     async def get_handled_count(self) -> int:
         return self._handled_count
-
-    @override
-    async def add_request(
```

Comment:
I'm pretty positive that removing `add_request` from `RequestList` is a good idea - the main purpose of the class is handling the `requestListSources` input from Apify anyway. Currently it is just an inefficient version of `RequestQueue`.

However, we may want to make a wrapper for `RequestSource` that just raises an error in `add_request`, so that crawlers that do not enqueue anything by design are possible. Thoughts?

Reply:
@vdusek a thumbs up is not enough here! :slightly_smiling_face: What part of the message do you agree with? If you think that a wrapper is a good idea, what would be a good name?